### PR TITLE
Adds information on using MPI in containers on ARCHER2

### LINKED ISSUE #204: Add information on creating and using Singularity containers with MPI
The Containers section does not currently have information on how to create and use containers with MPI on ARCHER2. This information does exist, see:

https://epcced.github.io/2020-12-08-Containers-Online/12-singularity-mpi/index.html

and the example ARCHER2 job submission script at:

https://github.com/EPCCed/2020-12-08-Containers-Online/blob/gh-pages/files/osu_latency.slurm.template

### PR DESCRIPTION
**Pull request by aturner-epcc**
Initial add of how to build and run Singularity containers with MPI on ARCHER2. Closes #204

### CODE REVIEW

**Review by mbareford — COMMENTED** [review]
That looks ok, but there are many places that have "!!! note", "!!! warning", "!!! tip" or "!!! info". Those tags are all being rendered as straight text in my browser.

File: docs/user-guide/containers.md [hunk 1]
```diff
@@ -169,37 +169,263 @@ with the job number.
...
+    Linux](https://www.sylabs.io/guides/2.6/user-guide/installation.html#install-on-linux)
+
+### Building Singularity images using Docker (macOS/Windows)
+
+!!! tip
+    You should install Docker Desktop to allow you to build Singularity
+    images using Docker. Instructions can be found at:
+     - [Installing Docker Desktop on Windows Professional](https://docs.docker.com/docker-for-windows/install/) 
+     - [Installing Docker Desktop on Windows Home](https://docs.docker.com/docker-for-windows/install-windows-home/) 
+     - [Installing Docker Desktop on macOS](https://docs.docker.com/docker-for-mac/install/)
```
Comment (juanfrh) [inline docs/user-guide/containers.md:199]: This admonition doesn't show bullet points (lines 194-199).

**Review by juanfrh — COMMENTED** [review]
I've built the documentation and I can see the admonitions well-formatted.

File: docs/user-guide/containers.md (hunk 1 above)
Comment (mbareford) [reply docs/user-guide/containers.md:199]: Thanks Juan, I must have something wrong with my setup. I'll go ahead and approve Andy's change.

**Review by mbareford — APPROVED** [review]
All looks fine.

**Pull request merged**